app/src/main/res/values/strings.xml

```diff
@@ -243,11 +243,11 @@
         <item>When tossing two fair dice there are 6*6 = 36 possible outcomes, all equally likely. The image is intended to hint at the solution without giving it away immediately: the circles are brighter when the sum of the two rolls is closer to seven.</item>
         <item>If everyone were gay the answer would be 0.9, while if everyone were straight the answer would be 0.1. In the situation we\'re thinking about, the answer is a weighted average of those two numbers. Do you understand why?</item>
         <item>A straightforward way to calculate the probability that one die beats another is to condition on the outcome of one of their rolls. For example, if the red die comes up 2, there\'s a 0/3 chance that it will beat blue; but if it comes up 4, it has a 1/3 chance. What if red comes up 9? Do you see how taking an average of those conditional probabilities will give you the correct answer?</item>
-        <item>A first hint is to show how life expectancy at birth was calculated: 0.2*(1 + 6 + 20 + 50 + 80) = 31.4. How can you modify that to get the answer we\'re looking for? Conditional on surviving to age 30, the probability of dying before age 30 must be zero. What about the (conditional) probability of dying between age 70 and 90?</item>
+        <item>A first hint is to show how life expectancy at birth was calculated: 0.2*(1 + 6 + 20 + 50 + 80) = 31.4. How can you modify that to get the answer we\'re looking for? Conditional on surviving to age 30, the probability of dying before age 30 must be zero. What about the (conditional) probability of dying between age 70 and 90? The numbers in this puzzle are made up, but they are intended to be semi-realistic -- see https://www.bbc.com/future/article/20181002-how-long-did-ancient-people-live-life-span-versus-longevity for a more detailed discussion of life spans in ancient Rome.</item>
```

Comment:
I don't see this additional information as a hint, it's more like background information. I do like it, though, to see where these puzzles or their ideas come from. Maybe it makes sense to add a separate (optional) button for this.

It might be even more elegant to have a region of text that can be collapsed and expanded, but I never implemented something like that on Android.

Reply:
Yes, you're right that this is strictly speaking not a hint, but additional background information, an invitation to do some further reading, etc. Perhaps it could be a hint in a very broad sense, if it helps build intuition or understanding. I've added information like this for a few other puzzles already, including https://github.com/atorch/probability_puzzles/blob/master/app/src/main/res/values/strings.xml#L238:

> see https://www.nytimes.com/2008/04/08/science/08tier.html and specifically the discussion following the sentence \"Dr. Chen says that the monkey’s distaste for blue can be completely explained with statistics alone\"